apps/web-giddh/src/app/import-excel/import-wizard/import-wizard.component.ts

```diff
@@ -75,9 +71,15 @@ export class ImportWizardComponent implements OnInit, OnDestroy {
     }
 
     public ngOnInit() {
-        this._activatedRoute.url.pipe(takeUntil(this.destroyed$)).subscribe(p => this.entity = p[0].path);
-        this.store.pipe(select(p => p.importExcel), takeUntil(this.destroyed$))
-            .subscribe(this.dataChanged);
+        this.activatedRoute.url.pipe(takeUntil(this.destroyed$)).subscribe(p => this.entity = p[0].path);
+
+        const importStatusRequest: ImportExcelStatusPaginatedResponse = new ImportExcelStatusPaginatedResponse();
+        importStatusRequest.totalItems = 0;
```

Comment:
Can we initialize `totalItems` as 0 in the constructor of `ImportExcelStatusPaginatedResponse`